docs/sources/datasources/loki/query-editor/index.md

```diff
@@ -81,17 +105,48 @@ Each operation's header displays its name, and additional action buttons appear
 | {{< figure src="/static/img/docs/v95/loki_operation_description.png" class="docs-image--no-shadow" max-width="30px" >}} | Opens the operation's description tooltip.                        |
 | {{< figure src="/static/img/docs/v95/loki_operation_remove.png" class="docs-image--no-shadow" max-width="30px" >}}      | Removes the operation.                                            |
 
-Some operations have additional parameters under the operation header.
-For details about each operation, use the `info` button to view the operation's description, or refer to the [Loki documentation](/docs/loki/latest/operations/).
+The query editor groups operations into the following sections:
 
-Some operations make sense only when used in a specific order.
-If adding an operation would result in nonsensical query, the query editor adds the operation to the correct place.
-To re-order operations manually, drag the operation box by its name and drop it into the desired place.
+- Aggregations - see [Built-in aggregation operators](/docs/loki/latest/logql/metric_queries/#built-in-aggregation-operators)
+- Range functions - see [Range Vector aggregation](/docs/loki/latest/logql/metric_queries/#range-vector-aggregation)
+- Formats - see [Log queries](/docs/loki/latest/logql/log_queries/#log-queries)
+- Binary operations - see [Binary operators](/docs/loki/latest/logql/#binary-operators)
+- Label filters - see [Label filter expression](/docs/loki/latest/logql/log_queries/#label-filter-expression)
+- Line filters - see [Line filter expression](/docs/loki/latest/logql/log_queries/#label-filter-expression)
 
-#### Hints
+Some operations make sense only when used in a specific order.b If adding an operation would result in nonsensical query, the query editor adds the operation to the correct place.
```

Comment:
```suggestion
Some operations make sense only when used in a specific order. If adding an operation would result in nonsensical query, the query editor adds the operation to the correct place.
```

Reply:
Does the query builder do this @ivanahuckova  ?

Reply:
Yes - you can test it out by creating query such as `{a="b"} | logfmt` in builder and adding line filter operation. It will be placed before the parser.